server/index.ts

```diff
@@ -34,10 +45,34 @@ app.use((req: Request, res: Response) => {
 
 app.use('/api', routes);
 
-app.get('*', (req: Request, res: Response) => {
+app.get('/', (req: Request, res: Response) => {
```

Comment:
I thought this needed to be changed because of the refresh causing a hanging load

Reply:
Yeah, I'm not sure. I get the hanging load with `*`  , `/`, and `/*`.